### DISCUSSION TITLE
How to install in HA?

### DISCUSSION THREAD

**Discussion by captain-ted** [Hyundai-Kia-Connect/kia_uvo#952] [Q&A] [answered]
I'm new to HA, but have installed other things today via HACS and manually. But I just can't figure out how to install this and there's no real instructions on the info. Please help! 😄

**Comment by cdnninja** [comment, marked as answer]
First you install via HACS.  After that you reboot home assistant.  Then add the integration just like you would any other integration. It will show a setup wizard to prompt username and passwords.

**Reply by captain-ted** [reply]
Thanks! Yeah for some reason it didn't show up but redid it and it did this time. I'm sure it was my user error - only set up HA today so much to learn.
